cmd/kubemark/hollow-node.go

```diff
@@ -86,6 +97,12 @@ func (c *hollowNodeConfig) addFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&c.UseRealProxier, "use-real-proxier", true, "Set to true if you want to use real proxier inside hollow-proxy.")
 	fs.DurationVar(&c.ProxierSyncPeriod, "proxier-sync-period", 30*time.Second, "Period that proxy rules are refreshed in hollow-proxy.")
 	fs.DurationVar(&c.ProxierMinSyncPeriod, "proxier-min-sync-period", 0, "Minimum period that proxy rules are refreshed in hollow-proxy.")
+
+	// for auth provider
+	fs.StringVar(&c.AuthProvider, "auth-provider", "", "The name of the auth provider (currently only supports 'eks').")
+	fs.StringVar(&c.ClusterContextName, "cluster-context-name", "", "The name of the cluster context, required to get CA from kubeconfig (required for eks auth provider).")
```

Comment:
Can you remove `required for eks auth provider` as I think the cluster-context is a very general thing